.github/workflows/deployment.yml

```diff
@@ -122,10 +85,10 @@ jobs:
           helm dependency update
           # Set the appVersion, workaround from https://github.com/helm/helm/issues/8194 so that we can
           # later find out which deployments need to be upgraded
-          sed -i 's/^appVersion:.*$/appVersion: "${{ steps.git-ref.outputs.sha }}"/' Chart.yaml
```

Comment:
Where did `steps.git-ref` come from previously? I can't find a step with this id anywhere :thinking: 
Must have been removed on an earlier refactoring somewhere, it's currently empty on Prod: https://app.ecamp3.ch/environment.js
But strange that the workflow never failed because of this...

Reply:
Yep, this was removed earlier but for some reason didn't fail